sdk/inc/azure/core/az_context.h

```diff
@@ -25,17 +26,15 @@
 /**
  * @brief A context is a node within a tree that represents expiration times and key/value pairs.
  *
- * @details The root node in the tree (ultimate parent) is #az_context_application which is a
```

Comment:
This comment still holds. I would keep it and just remove `Each new node is a child of some parent.`

Reply:
The comment doesn't make sense on the type.  How is every az_context the root node in the tree?

Reply:
The comment isn't talking about any/every context, just specifically `#az_context_application`

It's letting the caller know that if you want the ultimate root az_context, go grab that one.